css/reset.css

```diff
@@ -4,4 +4,6 @@
 
 body {
   margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(240, 240, 240);
```

Comment:
This should be moved to `styles.css`. The only styles that should be applied here are things related to "resetting a page" in CSS. Essentially removing the browsers default `margin` from the body, and setting the `box-sizing` to `border-box`